### PR TITLE
DSA-35-Find-the-minimum-and-maximum-element-in-an-array-using-Divide-and-Conquer

### PR DESCRIPTION
**Pull request by Umarfarook1**
Given an integer array, find the minimum and maximum element present in it by making minimum comparisons by using the divide-and-conquer technique. 

For example, 

Input: nums = [5, 7, 2, 4, 9, 6] Output: The minimum array element is 2The maximum array element is 9

### CODE REVIEW

File: DSA_Training/MinMax.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+def find_min_max(nums, low, high):
```
Comment (ganeshyevle) [inline DSA_Training/MinMax.py:2]: please add Comments here

File: DSA_Training/MinMax.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            return (nums[low], nums[high])
+        else:
+            return (nums[high], nums[low])
+    
+    mid = (low + high) // 2
+    min1, max1 = find_min_max(nums, low, mid)
+    min2, max2 = find_min_max(nums, mid + 1, high)
+    
+    return (min(min1, min2), max(max1, max2))
+
```
Comment (ganeshyevle) [inline DSA_Training/MinMax.py]: please add comment here

**Review by ganeshyevle — CHANGES_REQUESTED** [review]
Please Add proper Comments in your code

File: DSA_Training/MinMax.py (hunk 1 above)
Comment (Umarfarook1) [reply DSA_Training/MinMax.py:2]: Done

File: DSA_Training/MinMax.py (hunk 2 above)
Comment (Umarfarook1) [reply DSA_Training/MinMax.py]: done

**Comment by Umarfarook1** [comment]
> Please Add proper Comments in your code

Done

**Review by ganeshyevle — APPROVED** [review]

**Comment by ganeshyevle** [comment]
Done Approved

**Pull request merged**